src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -226,7 +235,12 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         {
             auto profileVM{ _viewModelForProfile(_settingsClone.ProfileDefaults()) };
             profileVM.IsBaseLayer(true);
-            contentFrame().Navigate(xaml_typename<Editor::Profiles>(), winrt::make<ProfilePageNavigationState>(profileVM, _settingsClone.GlobalSettings().ColorSchemes(), *this));
+
+            // Update the profiles navigation state
+            _profilesNavState.Profile(profileVM);
+            _profilesNavState.Schemes(_settingsClone.GlobalSettings().ColorSchemes());

```

Comment:
Do you even need to update the reference to `Schemes` here? Shouldn't we just update the profile?

Reply:
I honestly don't know. Before, we were taking a new reference to the ColorSchemes on each navigation - I bet we don't anymore